code/modules/awaymissions/gateway.dm

```diff
@@ -357,5 +357,8 @@ GLOBAL_LIST_EMPTY(gateway_destinations)
 	animate(get_filter("portal_ripple"), time = 1.3 SECONDS, loop = -1, easing = LINEAR_EASING, radius = 32)
 
 	var/turf/center_turf = our_destination.get_target_turf()
+	var/list/turfs_to_visualize = block(locate(center_turf.x - 1, center_turf.y - 1, center_turf.z), locate(center_turf.x + 1, center_turf.y + 1, center_turf.z))
+	for(var/turf/to_unset as anything in turfs_to_visualize)
+		to_unset.vis_flags &= ~VIS_HIDE
```

Comment:
This seems  a bit ick, is it the only way we can pull this off?

Reply:
Also what happens when a turf is changed inside the gateway's space?

Reply:
im not sure of another way to do it right now

Reply:
also its broken somehow here but not on master, possibly due to interference with how lighting works now

Reply:
all turfs no longer have the VIS_HIDE flag so this is no longer an issue